Dockerfile

```diff
@@ -28,4 +35,4 @@ WORKDIR /silcam
 RUN python setup.py develop
 
 # Run the PySilCam tests as default entrypoint
-ENTRYPOINT ["python", "setup.py", "test"]
+CMD ["python", "setup.py", "test"]
```

Comment:
Changed from ENTRYPOINT to CMD to allow the user of the container to override it. Makes it a bit easier to investigate issues locally.